examples/todoapp/README.md

```diff
@@ -1,6 +1,6 @@
-An example of Kotlin Multiplatform todo app with shared Jetpack Compose UI.
+An example of Kotlin Multiplatform todo app with shared Android/Desktop Compose UI and SwiftUI iOS.
 
-Supported targets: `Android` and `JVM`.
+Supported targets: `Android`, `JVM` and `iOS`
```

Comment:
Supported targets: Android (Compose), JVM (Compose) and iOS (SwiftUI, not Compose)